src/instantsearch.js

```diff
@@ -2,4 +2,4 @@ export { createSuitMixin } from './mixins/suit';
 export { createWidgetMixin } from './mixins/widget';
 export * from './widgets';
 export { plugin as default } from './plugin';
-export { createInstantSearch } from './util/createInstantSearch';
+export { createServerRootMixin } from './util/createServerRootMixin';
```

Comment:
could use a second opinion on this API, we can also go with like before: 

```
const { instantsearch, rootMixin } = createInstantSearch()
```

(now it's)

```
const rootMixin = createInstantSearch()
```

Reply:
You mean, now it's `const rootMixin = createServerRootMixin()`, right?

Since the new function returns only a mixin and we don't need an instace of InstantSearch there, so `createServerRootMixin` sounds proper to me.

Reply:
Yes, the difference is that if we would change the signature to be just like before again, you could access the InstantSearch variables without them being put inside data. I'm not sure there's an advantage for either point though